src/Components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import './Tabs.scss';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
```

Comment:
this interface duplicates in both files. Create it somewhere and use it in both places.